theme-settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * theme-settings.php
+ */
+
+/**
+ * Implements hook_form_system_theme_settings_alter().
+ */
+function europa_form_system_theme_settings_alter(&$form, &$form_state) {
```

Comment:
@nelsongrin, that hook in only called on the theme setting, so i don't understand this request:
https://api.drupal.org/api/drupal/modules%21system%21theme.api.php/function/hook_form_system_theme_settings_alter/7.x